clash-lib/src/Clash/Primitives/DSL.hs

```diff
@@ -433,9 +438,13 @@ enableToBit bitName = \case
         [ (Just (BoolLit True), Literal Nothing (BitLit H))
         , (Nothing            , Literal Nothing (BitLit L))
         ]
+    declareUseOnce (Proc Blocking) uniqueBitName
     pure texp
   tExpr -> error $ "enableToBit: Got \"" <> show tExpr <> "\" expected Enable"
 
+declareUseOnce :: HasUsageMap s => Usage -> Identifier -> State s ()
+declareUseOnce u i = usageMap %= Map.insert (Id.toText i) u
```

Comment:
This should be moved next to `declareUse`. Could you document the differences?

Reply:
Also, with the "once" in the name I'd expect it to crash if there was already an identifier with the same name in the map.

Reply:
Oh fair enough!